### DISCUSSION TITLE
Does Frigate+ use downscaled images for training?

### DISCUSSION THREAD

**Discussion by nuansed** [blakeblackshear/frigate#3416] [Ask A Question]
Hi, I've uploaded a few 3840x2160 pixel images to Frigate+, but they all seem to have been downsized to a 640x360 (also looks like a nearest neighbour algo caused lots of artefacts). Is the downsized image just a low res preview to reduce bandwidth usage while the full-sized image is utilised in the backend? 

**Example**
Cropped original image:
![Sequence 01 00_01_12_06 Still056 copy 2](https://user-images.githubusercontent.com/12543435/177028679-39f58ad8-200d-493f-a32a-502e7ca9b1fd.jpg)

Cropped image from https://plus.frigate.video/dashboard/edit-image/?id=...
![01G7172XEZWE7AZ0PW268FFTVE-annotate copy](https://user-images.githubusercontent.com/12543435/177028674-9cfef35c-01f8-4dd8-bb36-46384bb5a1a5.jpg)

**Comment by NickM-27** [comment]
Yes you've got it, annotations are served on a lower res version to conserve bandwidth, the actual training is done on higher res image

**Reply by nuansed** [reply]
Ah ok, good to know. It'd be helpful to have less artefacts, maybe with a bilinear resize. 

It's also a bit difficult to accurately set the bounding box on the small subject above. How about resizing images to 30% of their original size instead? Maybe with a minimum of 640x360?
